ref/general/authorization.adoc

```diff
@@ -17,7 +17,13 @@ In order for an application to be secure, it must be able to manage access to pr
 
 In an application where only certain users or groups are entitled to access certain resources, users and groups are assigned roles that represent which resources they can access. For example, in a payroll application, users in the `Employee` role can view a page that lists all employees and their contact information. However, only users in the `Manager` role can view information about how much each employee is paid. During link:/docs/ref/general/#authentication.html[authentication], an application determines who a user is and what roles they are in. During authorization, Open Liberty determines how these roles map to different protected resources on the server.
 
-Typically, a developer can specify which roles are allowed to access an application resource directly in the source code by setting the `RolesAllowed` annotation to contain the permissible roles. When a user is authenticated, the access groups to which the user is assigned are retrieved from a user registry and added to the `Subject` object for that user. The Open Liberty server maps these groups to the roles that defined in the application. If the role or roles that are mapped to the user match the roles that are specified for the resource, the user is granted access.
+The roles that users and groups are checked against can either be defined directly in the application or applied to the application by way of the server configuration.
+Roles are defined directly in the application by setting the `@DeclareRoles` annotation or by using `<security-role>` elements in the `web.xml` deployment descriptor file.
+To apply roles to an application from the server configuration, specify the `<security-role>` element within the `<application-bnd>` element in the `server.xml` file.
+
+In some use cases, an application needs to know only whether a user is authenticated or not. In such cases, configuration can be simplified by using the `"**"` special value in the `@rolesAllowed` annotation or XML configuration. This value denotes that any authenticated user can access a resource. It is also possible to effectively bypass authorization altogether by using the `@PermitAll` annotation, which specifies that a resource is accessible to everyone, even users who are not authenticated.
+
+Typically, a developer can specify which roles are allowed to access an application resource directly in the source code by setting the `RolesAllowed` annotation to contain the permissible roles. When a user is authenticated, the access groups to which the user is assigned are retrieved from a user registry and added to the `Subject` object for that user. The Open Liberty server maps these groups to the roles that are defined in the application. If the role or roles that are mapped to the user match the roles that are specified for the resource, the user is granted access.
```

Comment:
Peer review feedback (while the PR is open): 

- Need to have consistency in the `RolesAllowed` annotation. (I think it should be `@RolesAllowed` annotation, maybe.) 
- Capitalize the `P` in `MicroProfile` in the last sentence of the body of the topic. 
- Switch `the` and `in` in `...in the application or in application-bnd stanzas the in `server.xml` file.`
- I don't think `Authorization` should be capitalized in `Ready to try out Authorization with Open Liberty for yourself?`

Other than that, looks good!!

Reply:
Thanks @Charlotte-Holt - these issues fixed in new commit